src/main/java/ca/corefacility/bioinformatics/irida/web/controller/api/samples/RESTSampleAssemblyController.java

```diff
@@ -182,8 +183,7 @@ public ModelMap addNewAssemblyToSample(@PathVariable Long sampleId, @RequestPart
 		} finally {
 			// clean up the temporary files.
 			logger.trace("Deleted temp files");
-			Files.deleteIfExists(target);
-			Files.deleteIfExists(temp);
+			IridaFiles.cleanupLocalTemporaryFiles(target, temp);
```

Comment:
As discussed lets revert these changes back to the `deleteIfExists` stuff for now.  We can refactor this in a later branch when needed.

Reply:
Reverted in 1dbdb47af3eedeee57410916380b562e92a85c41